bin/i18n/i18n_script_utils.rb

```diff
@@ -364,7 +323,7 @@ def self.file_changed?(locale, file)
 
   # Formats strings like 'en-US' to 'en_us'
   #
-  # @param [String] locale the BCP 47 (IETF language tag) format (e.g., 'en-US')
+  # @param locale [String] the BCP 47 (IETF language tag) format (e.g., 'en-US')
```

Comment:
Better format here, thank you.

Reply:
Yep. my mistake. https://rubydoc.info/gems/yard/file/docs/GettingStarted.md